pcsx2/GS/Renderers/Common/GSRenderer.cpp

```diff
@@ -590,7 +595,7 @@ void GSRenderer::VSync(u32 field, bool registers_written, bool idle_frame)
 
 	// Little bit ugly, but we can't do CAS inside the render pass.
 	GSVector4i src_rect;
-	GSVector4 src_uv, draw_rect;
+	GSVector4 src_uv = {}, draw_rect = {};
```

Comment:
These were fine, and any warnings popping up from a compiler/static analysis is a false positive. All paths which use the variables have them assigned.